contracts/governance/FBeetsRevenueSharer.sol

```diff
@@ -72,21 +76,27 @@ contract FBeetsRevenueSharer is
         _distribute();
     }
 
+    function setFBeetsLockerShare(uint256 newFBeetsLockerShare) external onlyRole(OPERATOR_ROLE){
+        fBeetsLockerShare = newFBeetsLockerShare;
+    }
+
     function _distribute() internal {
         if (beets.balanceOf(address(this)) > 0) {
-            // 50% of the rewards go to locked fBeets holders
-            uint256 rewardsForLockedFBeets = beets.balanceOf(address(this)) / 2;
+            // fBeetsLockerShare of the rewards go to locked fBeets holders
+            uint256 rewardsForLockedFBeets = beets.balanceOf(address(this)) * fBeetsLockerShare / DENOMINATOR;
             beets.approve(address(locker), rewardsForLockedFBeets);
             locker.notifyRewardAmount(address(beets), rewardsForLockedFBeets);
 
-            // the other 50% we convert to fBeets and share with all fBeets holders
+            // the rest we convert to fBeets and share with all fBeets holders
             // we cannot assign fix size arrays to dynamic arrays, so we do this thing...
             address[] memory assets = new address[](1);
             assets[0] = address(beets);
             uint256[] memory amountsIn = new uint256[](1);
             amountsIn[0] = beets.balanceOf(address(this));
             uint256 minBptOut = 0;
 
+            require(amountsIn[0] > 0, "No share for non-lockers left");
```

Comment:
so this would revert in case 100% goes to lockers which is not exactly what we want :) so we rather want it in an `if` statement. ideally above line 92 so we dont waste gas allocating memory for the arrays we dont need afterwards

Reply:
lol, right. Changed it.